bin/travis/setupphp.sh

```diff
@@ -2,4 +2,4 @@
 
 add-apt-repository -y ppa:ondrej/php5
 apt-get update -q
-apt-get install -y php5-cli php5-common php5-gd php5-xsl php5-intl
+sudo apt-get install -q -y --force-yes php5-cli php5-common php5-gd php5-xsl php5-intl
```

Comment:
you don't need to add sudo here as the script is already executed with sudo in .travil.yml and in prepare_behat.sh.